pkg/transfer/transfer.go

```diff
@@ -60,7 +60,7 @@ type ImageFilterer interface {
 // ImageStorer is a type which is capable of storing an image to
```

Comment:
minor nit: this comment may not need to be changed to reflect the multiple references capability, but it is currently not properly readable? "storing an image to for a provided descriptor"? Maybe it was supposed to say "store an image to a specific image store for a provided descriptor"?